plugins/CoreHome/angularjs/history/history.service.js

```diff
@@ -67,7 +67,12 @@
                     searchObject[name] = searchObject[name][searchObject[name].length - 1];
                 }
 
-                searchString.push(name + '=' + encodeURIComponent(searchObject[name]));
+                var value = searchObject[name];
+                if (name != 'columns') { // the columns query parameter is not urldecoded in PHP code. TODO: this should be fixed in 3.0
```

Comment:
@diosmosis maybe you can create follow up issue for 3.0.0 and, ideally, a pull request while this is fresh in your mind? :-)